### DISCUSSION TITLE
Make osu! avalible on F-Droid store page.

### DISCUSSION THREAD

**Discussion by MeuhMeuh62** [ppy/osu#30422] [Ideas] [answered]
It's been a while i play osu! on my phone. Is it possible to publish on the F-Droid store? For better updates? Idk.
I checked the F-Droid website and i screenshot the Submit applications area.
![Screenshot_20241025-110031.png](https://github.com/user-attachments/assets/f1049656-84c0-4303-bf0b-70ab002e615b)
I seen and it's can be something eazy.
https://f-droid.org/en/contribute/

**Comment by Gabixel** [comment, marked as answer]
[Duplicate](https://github.com/ppy/osu/discussions?discussions_q=f+droid+). Not sure if they are actively looking for mobile app stores to publish to, according to [this answer](https://github.com/ppy/osu/discussions/13593#discussioncomment-894080), but I guess it will happen one day
